jerry-debugger/jerry_client_main.py

```diff
@@ -885,13 +887,13 @@ def _parse_source(self, data):
                 return ""
 
             elif buffer_type in [JERRY_DEBUGGER_SOURCE_CODE, JERRY_DEBUGGER_SOURCE_CODE_END]:
-                source_code += data[1:]
+                source_code += data[1:].decode('latin1')
 
             elif buffer_type in [JERRY_DEBUGGER_SOURCE_CODE_NAME, JERRY_DEBUGGER_SOURCE_CODE_NAME_END]:
-                source_code_name += data[1:]
+                source_code_name += data[1:].decode('latin1')
```

Comment:
Same question here for source code names.

Reply:
Yes utf-8 should be used.

same reason as here: https://github.com/jerryscript-project/jerryscript/pull/5095#discussion_r1330545099